torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4353,6 +4377,100 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.mode,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_mode,),
+    UnaryUfuncInfo('mvlgamma',
+                   ref=reference_mvlgamma if TEST_SCIPY else _NOTHING,
```

Comment:
Creating multiple OpInfos for mvlgamma makes sense and I appreciate this PRs thoughtfulness and attention to detail. I wish there were a better way to represent mvlgamma, though. I guess operations where the domain depends on the input are just very rare?